README.md

```diff
@@ -1,6 +1,27 @@
 # jubmoji.quest
 
-Website v2 for use at Zuconnect &amp; Devconnect
+## Overview
+
+Users of Jubmoji.quest tap NFC cards to collect signatures. By collecting these signatures, they complete quests. In order to complete a quest, a user must generate a client-side zero knowledge proof about the signatures they have collected. In particular, we allow users to complete quests which will send their proof to an external server for custom verification.
+
+## What is a Jubmoji?
+
+We have distributed 201 NFC Cards across Zuconnect/Devconnect. Each of these cards has a [BabyJubJub](https://eips.ethereum.org/EIPS/eip-2494) ECDSA private key stored in its trusted hardware, a list of all public keys can be found [here](https://github.com/jubmoji/jubmoji.quest/blob/main/packages/jubmoji-api/src/data/cardPubKeys.ts). When a user taps an NFC Card, the card increments a nonce and signs the pair `(nonce, randomSalt)` with its private key. This signature is the Jubmoji.
```

Comment:
Should maybe reference libhalo here? Specifically https://github.com/arx-research/libhalo/blob/master/docs/halo-command-set.md#command-sign_random